src/etc/header.txt

```diff
@@ -0,0 +1,11 @@
+Copyright 2015 The OpenTracing Authors
```

Comment:
"The OpenTracing Authors" is a way out of enumerating everyone, so equally nice to all. I use this approach in many projects, and got the idea from guava.